packages/cli/src/cmds/beacon/initPeerIdAndEnr.ts

```diff
@@ -87,10 +86,6 @@ export function overwriteEnrWithCliArgs(
     maybeUpdateEnr(enr, "tcp6", args["enr.tcp6"] ?? port6 ?? enr.tcp6);
   }
 
-  // cgc is big ending but since 1 bytes suffices for now so its the same
-  const cgc = Math.max(config.CUSTODY_REQUIREMENT, config.NODE_CUSTODY_REQUIREMENT);
-  enr.set("cgc", intToBytes(cgc, Math.ceil(Math.log2(cgc + 1) / 8), "be"));
```

Comment:
This was causing the test for persisting the ENR to disk to fail -- it was undefined on first write, then defined to this on second write.

I don't think this is needed here anyway, since the CGC value is already set through the custody config, which flows through the `MetadataController` into the discV5 ENR. Additionally, even if we wanted to propagate the most recent CGC value to the ENR disk file whenever it changes, I don't think we'd have the context to do it from here.

Reply:
I tend to agree.  I don't think we need to persist `cgc` to disk at all because its dynamic now.